src/opm/parser/eclipse/EclipseState/IOConfig/RestartConfig.cpp

```diff
@@ -332,17 +333,16 @@ inline std::map< std::string, int > RPT( const DeckKeyword& keyword,
 
     if (ints && strs) {
         const auto& location = keyword.location();
-        std::string msg = "Mixed style input is not allowed for keyword: " + keyword.name() + " at " + location.filename + "(" + std::to_string( location.lineno ) + ")";
-        parseContext.handleError(ParseContext::RPT_MIXED_STYLE, msg, errors);
+        std::string msg = "Error in keyword {keyword}, mixing mnemonics and integers is not allowed\n"
```

Comment:
I have to say that I find the new style here, with keyword format args, quite readable. I agree with Alf that this looks nice. I think we mostly worked out the disagreements in the discussion about the exception class.

However, you claim that these messages are consistent in some sense. Is that simply by convention? I do not see any common code between these messages? By convention, I mean it looks like many messages end with:
```
... (end-of-line)
"In {file} line {line}."
```

Reply:
> Is that simply by convention? 

yes - it is simply by convention. I agree that maybe that convention should be codified - but I I wanted to get a bit more experience/feel for it before I created a straightjacket for this.